### ISSUE TITLE
Expose the entire secret contents?

### ISSUE THREAD

**Issue by phillipj** [camptocamp/terraform-provider-pass#8]
Just learned a bunch about gopass while debugging my way trying to get a hold of the *entire* secret contents. A couple of things surprised me as I'm obviously no pass/gopass expert;

- `.password`: the first line of the secret
- `.body`: remaining lines of the secret

I've got a RSA private key in my pass store that I want to read. That secret contains the whole private key and nothing more, e.g:

```
-----BEGIN RSA PRIVATE KEY-----

...

-----END RSA PRIVATE KEY-----
```

I'd really appreciate if I could get a hold of the entire contents in that secret, without having to concatenate `password + body` everywhere.. Found that gopass exposes this as [`Secret.String()`](https://github.com/justwatchcom/gopass/blob/master/pkg/store/secret/secret.go#L66).

Threw a test together locally and that put that into the `.body` attribute instead and it worked as expected for my use case.

Got any thoughts about exposing that somehow, e.g. a new `string` attribute on the [data source](https://github.com/camptocamp/terraform-provider-pass/blob/master/pass/data_source_password.go#L16)?

**Comment by espoelstra** [comment]
At least from the CLI you can omit `-c` to `gopass show some/secret` to get the whole thing, I like what you've done with `.full` for this provider though.

**Comment by mcanevet** [comment]
Fixed by #9
